Head-of-Product.md

```diff
@@ -0,0 +1,62 @@
+## About [Snyk](https://snyk.io)
+
+<img src="https://snyk.io/images/snyk-dog.png" width="100" alt="Snyk logo" />
+
+Snyk's mission is to help developers use open source code and stay secure. 
+Open Source code, pulled from npm, Maven, RubyGems and others, is often the majority of code in an application, and yet it receives no security attention. 
+Most developers do nothing to track known vulnerabilities in the packages they use, and trust this code as though its their own. 
+This risk is massive today, and is only made worse by growing use of packages and trends such as Serverless and PaaS, who secure the server's dependencies but leave the app to fend with its dependencies security alone.
+
+At Snyk we believe this problem can only be fixed by developers and DevOps teams, and built into the way we create software. 
+We build a SaaS CI/CD and runtime monitoring product that helps organisations continuosly find and fix vulnerable dependencies without slowing down.
```

Comment:
I don't believe 'runtime' would be understood like you mean it to by someone not from within